ginkgo.Jenkinsfile

```diff
@@ -57,10 +67,10 @@ pipeline {
                         sh 'cd ${TESTDIR}; ginkgo --focus="RuntimeValidated*" -v -noColor'
                     },
                     "K8s-1.9":{
-                        sh 'cd ${TESTDIR}; K8S_VERSION=1.9 ginkgo --focus=" K8sValidated*" -v -noColor'
+                        sh 'cd ${TESTDIR}; K8S_VERSION=1.9 ginkgo --focus=" K8sValidated*" -v -noColor ${FAILFAST}'
```

Comment:
Have you validated whether the `AfterEach` blocks get executed if a test fails with `-failFast` enabled? I want to make sure we have logs to triage / examine / peruse if builds fail with this option enabled.

Reply:
Yep, `AfterEach` is called:

Example: 
```
package gink_example_test

import (
	"fmt"
	"testing"

	. "github.com/eloycoto/ginkgo-ext"
	"github.com/onsi/ginkgo/reporters"

	"github.com/onsi/ginkgo"
	. "github.com/onsi/gomega"
)

func TestGinkExample(t *testing.T) {
	RegisterFailHandler(Fail)
	junitReporter := reporters.NewJUnitReporter("junit.xml")
	RunSpecsWithDefaultAndCustomReporters(t, "Foo Suite", []ginkgo.Reporter{junitReporter})
}

var _ = Describe("K8Spolicies", func() {
	AfterEach(func() {
		fmt.Println("----------------------")
		fmt.Println("----------------------")
		fmt.Println("----------------------")
		fmt.Println("AfterEach")
		fmt.Println("----------------------")
		fmt.Println("----------------------")
		fmt.Println("----------------------")
	})

	It("Test1", func() {
		Expect(true).To(BeFalse())
	})
})
```

```
----------------------                 
----------------------                 
----------------------                 
AfterEach                              
----------------------                 
----------------------                 
----------------------                 
• Failure [0.001 seconds]              
K8Spolicies                            
/home/ecoto/.go/src/github.com/eloycoto/ginkgo-ext/scopes.go:170               
  Test1 [It]                           
  /home/ecoto/.go/src/github.com/eloycoto/ginkgo-ext/scopes.go:202             

  Expected                             
      <bool>: true                     
  to be false                          

  /home/ecoto/.go/src/github.com/eloycoto/gink_example/gink_example_suite_test.go:32           
------------------------------         


Summarizing 1 Failure:                 

[Fail] K8Spolicies [It] Test1          
/home/ecoto/.go/src/github.com/eloycoto/gink_example/gink_example_suite_test.go:32             

Ran 1 of 1 Specs in 0.001 seconds      
FAIL! -- 0 Passed | 1 Failed | 0 Pending | 0 Skipped --- FAIL: TestGinkExample (0.00s)         
FAIL                                   
```